app/views/admin/users/_new.html.haml

```diff
@@ -0,0 +1,20 @@
+- content_for :modal_content do
+    = form_with url: admin_users_path, scope: :user, class: 'w-full max-w-md' do |form|
+        %div{class: 'flex flex-wrap -mx-3 mb-6'}
```

Comment:
this could be written like this
```
.flex.flex-wrap.-mx-3.mb-6
  .w-full.px-3.mb-6.md:mb-0
    = form.label :email, class:  'ca-form-label'
    = form.email_field :email, class: 'ca-form-input-field'
```
And you could put this `.flex.flex-wrap.-mx-3.mb-6` in one class and move it to tailwind.scss